app-devel/git-delta/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.18.2
+SRCS="git::commit=tags/$VER::https://github.com/dandavison/delta.git"
```

Comment:
This package should be named `delta` with a `PKGPROV` to `git-delta`. We follow upstream project names by principle.